src/SFA.DAS.EmployerDemand.Web.AcceptanceTests/Steps/HttpSteps.cs

```diff
@@ -57,7 +58,35 @@ public void ThenAnHttpStatusCodeIsReturned(int httpStatusCode)
 
             result.StatusCode.Should().Be(httpStatusCode);
         }
-        
-    }
+
+        [When("I navigate to location header url")]
+        public async Task WhenINavigateToTheLocationHeaderUrl()
+        {
+            if (!_context.TryGetValue<HttpResponseMessage>(ContextKeys.HttpResponse, out var result))
+            {
+                Assert.Fail($"scenario context does not contain value for key [{ContextKeys.HttpResponse}]");
+            }
+
+            var client = _context.Get<HttpClient>(ContextKeys.HttpClient);
+            var continuedResponse = await client.GetAsync(result.Headers.Location);
+            _context.Set(continuedResponse, ContextKeys.HttpResponse);
+        }
+
+        [When(@"I post the viewed page")]
+        public async Task WhenIPostTheViewedPageAsync()
+        {
+            if (!_context.TryGetValue<HttpResponseMessage>(ContextKeys.HttpResponse, out var result))
+            {
+                Assert.Fail($"scenario context does not contain value for key [{ContextKeys.HttpResponse}]");
+            }
+
+            var client = _context.Get<HttpClient>(ContextKeys.HttpClient);
+            HttpContent content = null;
+            var continuedResponse = await client.PostAsync(result.RequestMessage.RequestUri.PathAndQuery, content);

```

Comment:
Can remove the `HttpContent content = null;` and just pass null in the PostAsync

```suggestion
            var continuedResponse = await client.PostAsync(result.RequestMessage.RequestUri.PathAndQuery, null);

```